### PR TITLE
Updated RModelParser_PyTorch.cxx -Fixes the bug related to build

### PR DESCRIPTION
**Pull request by Neel-Shah-29**
# This Pull request: Fixes the bug to build ROOT in Ubuntu 20.04
image

The above bug was noticed by me when i was building boost in my local computer. This can be resolved by just removing example_outputs argument from PyRunString function over [here](https://github.com/root-project/root/blob/master/tmva/pymva/src/RModelParser_PyTorch.cxx#L422)

## Changes or fixes:
Bug-issue related to build

## Checklist:

- [x] tested changes locally
- [ ] updated the docs (if necessary)

### CODE REVIEW

**Review by lmoneta — COMMENTED** [review]
Which PyTorch version are you using ? 
With 1.10 it still works for me, but I think looking at 
https://github.com/pytorch/pytorch/blob/master/torch/onnx/utils.py 
the master version has the _model_to_graph interface changed to not pass anymore example_outputs. 
@sanjibansg , can we remove example_outputs also for older PyTorch versions ?

File: tmva/pymva/src/RModelParser_PyTorch.cxx [hunk 1]
```diff
@@ -419,7 +419,7 @@ RModel Parse(std::string filename, std::vector<std::vector<size_t>> inputShapes,
```
[the remark this answers is not in the document]
Comment (sanjibansg) [reply tmva/pymva/src/RModelParser_PyTorch.cxx:419]: It was deprecated in v1.10 and was completely removed with this PR https://github.com/pytorch/pytorch/pull/62815 for v1.11, but I think it was a required argument in the earlier versions.

**Review by sanjibansg — CHANGES_REQUESTED** [review]
I think if we are restricting to use the current version of PyTorch with example_outputs removed, then this line can be removed as well.
https://github.com/root-project/root/pull/10223/files/cb144c0f3939dd5a357fe9cba748d1d44ed395cd#diff-358c76d81a2584d32938feea96f50f0ef04b3ef051a8adc124847a1ff5d29743R418

File: tmva/pymva/src/RModelParser_PyTorch.cxx (hunk 1 above)
[the remark this answers is not in the document]
Comment (lmoneta) [reply tmva/pymva/src/RModelParser_PyTorch.cxx:419]: Thanks Sanjibang. I will check the PR with an older PyTorch version

**Comment by Neel-Shah-29** [comment]
> Which PyTorch version are you using ? With 1.10 it still works for me, but I think looking at https://github.com/pytorch/pytorch/blob/master/torch/onnx/utils.py the master version has the _model_to_graph interface changed to not pass anymore example_outputs. @sanjibansg , can we remove example_outputs also for older PyTorch versions ?

image
Hello, My Pytorch version is 1.11 and I think for 1.10 it works fine but for higher versions it gives the above error i will make the changes proposed by @sanjibansg and @lmoneta . Thankyou for reviewing my PR.

File: tmva/pymva/src/RModelParser_PyTorch.cxx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Neel-Shah-29) [reply tmva/pymva/src/RModelParser_PyTorch.cxx:417]: I have removed the line 418 as suggested. Thankyou.

**Comment by Neel-Shah-29** [comment]
Hello @lmoneta can you merge this PR if everything is alright?

**Review by lmoneta — APPROVED** [review]
LGTM!
Thank you for this fix!

**Pull request merged**